openedx_learning/core/publishing/api.py

```diff
@@ -107,6 +162,37 @@ def learning_package_exists(key: str) -> bool:
     return LearningPackage.objects.filter(key=key).exists()
 
 
+def get_last_publish(learning_package_id: int) -> PublishLog | None:
+    return PublishLog.objects \
+                     .filter(learning_package_id=learning_package_id) \
+                     .order_by('-id') \
+                     .first()
+
+def get_all_drafts(learning_package_id: int):
```

Comment:
Nit: I think you can set these return types to `QuerySet[Draft]` etc., as long as you have `from __future__ import annotations`